osfoffline/application/background.py

```diff
@@ -55,6 +55,8 @@ def run(self):
         self.user = self.get_current_user()
         self.osf_folder = self.user.osf_local_folder_path
 
+        LocalDBSync(self.observer, self.user, self.loop).sync_db()
```

Comment:
(comment duplicated from #55)

This is probably not the best place to call `LocalDBSync`. We want to use initial sync only when the program first starts, but putting here means it may also be called when we click sync now, when the prefs panel is opened/closed, etc. This may create a situation where the "initial" sync (this class) generates events that do not match "active sync" (watchdog file monitoring, eg move and deletes).

We will need to test carefully and revisit best place for this code once the program restructuring is done. After merge, it may make sense to turn off `LocalDBSync` for now. (keep the refactored logic, but don't use it yet)

This line appears in BackgroundWorker.run(...), which is called when the thread is started. Eg:
1. OsfApp.start (which is our intended goal)
2. OsfApp.resume (prefs panel close, syc now, etc)